index.html

```diff
@@ -5061,6 +5061,143 @@ <h2 id="AudioContextInfo">
+                    </li>
+                    <li>
+                      <code><em>defaultValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="defaultValue")</code>.
+                    </li>
+                    <li>
+                      <code><em>minValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="minValue")</code>.
+                    </li>
+                    <li>
+                      <code><em>maxValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="maxValue")</code>.
+                    </li>
+                  </ul>
+                </li>
+                <li>Let <code><em>initialValue</em></code> be the result of
+                <code><a href=
+                "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=<em>options</em>,
+                P=<em>paramName</em>)</code>.
+                  <ul>
+                    <li>If the result of <a href=
+                    "https://tc39.github.io/ecma262/#sec-ecmascript-data-types-and-values">
+                      Type</a><code>(<em>initialValue</em>)</code> is
+                      <code>Number</code>, perform <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-set-o-p-v-throw">Set</a>(O=<em>audioParam</em>,
+                      P="value", V=<em>initialValue</em>)</code>. Otherwise
+                      ignore and go to the next step.
+                    </li>
+                  </ul>
+                </li>
+                <li>Perform <code><a href=
+                "https://tc39.github.io/ecma262/#sec-set-o-p-v-throw">Set</a>(O=<em>baseNodeInstance</em>,
+                P=<em>paramName</em>, V=<em>audioParam</em>, false)</code>.
+                </li>
+              </ol>
+            </li>
+            <li>Let <code><em>nodeInstance</em></code> be the result of
+              <a href="http://www.ecma-international.org/ecma-262/6.0/#sec-construct">
+              Construct</a><code>(<a>AudioWorkletNode</a>, <em>options</em>,
+              <em>baseNodeInstance</em>)</code>.
+              <ul>
+                <li>If the constructor throws an exception, set the
+                definition’s constructor valid flag to <code>false</code>,
+                abort all these steps.
+                </li>
+              </ul>
+            </li>
+            <li>Create the internal association between
+            <code><em>nodeInstance</em></code> and
+            <code><em>processorInstance</em></code>. Note that these are not
+            exposed to users:
+              <ol>
+                <li>Set <code><em>processorReference</em></code> in <code><em>
```

Comment:
processorReference and nodeReference are not discussed anywhere else. Probably they are part of internal state of the objects. In which case, in the definition of AudioWorkletNode, you should say something like "Every AudioWorkletNode has an associated **processor reference**, initially null." and then cross-reference it here. (You should not use code font for it, since it is not user-exposed.)